### PR TITLE
feat(docs): new home page

### PR DESCRIPTION
**Pull request by vimwitch**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally in root directory and any changes were pushed
- [x] Lint (`yarn lint:fix`) was run locally in root directory and any changes were pushed

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the new behavior?

Preview of the new homepage [here](  https://unirep-docs-test.jchancehud.workers.dev)

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by vimwitch** [comment]
@ChialiT New homepage built. The header/footer are left as they were, I'm not sure how to customize it with docusaurus as specified in the designs.

Preview [here](https://unirep-docs-test.jchancehud.workers.dev/). The link in the "Use UniRep to build a Reddit clone" section doesn't go anywhere, not sure where it should go.

**Comment by vimwitch** [comment]
Also the font sizes in the figma file are too big for mobile screens to render.

File: packages/docs/src/pages/index.js [hunk 1]
```diff
@@ -9,55 +9,292 @@ import { Hero } from '@site/src/components/Hero'
...
+                        description="Trustless interoperability fosters innovation and upholds user autonomy— data is only revealed or altered with the user's consent."
+                    />
+                </div>
+                <div style={{ height: '56px' }} />
+                <div style={{ alignSelf: 'flex-start' }}>
+                    <Button
+                        style={{ backgroundColor: 'white', color: 'black' }}
+                    >
+                        Learn more
+                    </Button>
```
Comment (vivianjeng) [inline packages/docs/src/pages/index.js:152]: this button doesn't work

File: packages/docs/src/pages/index.js [hunk 2]
```diff
@@ -9,55 +9,292 @@ import { Hero } from '@site/src/components/Hero'
...
+                }}
+            >
+                A protocol built for handling user data anonymously.
+            </div>
+            <div style={{ height: '24px' }} />
+            <div className={styles.paragraph} style={{ maxWidth: '768px' }}>
+                UniRep is a Zero-Knowledge Protocol for user data & reputation
+                management. We use pioneering technology to offer a space for
+                developers and users alike to explore the potential of
+                privacy-centered online interactions.
```
Comment (vivianjeng) [inline packages/docs/src/pages/index.js:99]: @ChialiT I think we should talk about how UniRep can do 
e.g. Users of UniRep can receive data anonymously and prove data with the user's consent.

in this paragraph I think it is hard to understand what it is different from other zk applications and why we use this protocol.

File: packages/docs/src/pages/index.js [hunk 3]
```diff
@@ -9,55 +9,292 @@ import { Hero } from '@site/src/components/Hero'
...
+            >
+                <div
+                    style={{
+                        color: 'black',
+                        fontSize: '56px',
+                        fontWeight: 700,
+                        lineHeight: '53px',
+                    }}
+                >
+                    Use UniRep to build a Reddit clone
```
Comment (vivianjeng) [inline packages/docs/src/pages/index.js:124]: @ChialiT 
I think only mentioning **Reddit** is too limited

Comment (ChialiT) [reply packages/docs/src/pages/index.js]: this is not the intention :)

**Review by ChialiT — APPROVED** [review]
This looks good for me, thank you so much for helping close this task :) @CJ-Rose

**Pull request merged**